src/clic.adoc

```diff
@@ -525,79 +527,12 @@ When {sithreshold} is a nonzero value P,
 interrupt sources with priority numbers P and higher behave as though those sources were not enabled,
 regardless of the settings of their interrupt-enable bits.
 
-== Same Privilege Mode Interrupt Preemption Support - smclic
+== Horizontal Nested Interrupt Preemption support for M-mode - smclic
 
-The Smclic extension depends on the Smclicincr extension.
+The Smclic extension depends on the Smaia extension.

```

Comment:
I guess this dependency is only to pull in these CSRs, and doesn't require implementing the rest of the AIA?
![bilde](https://github.com/user-attachments/assets/46b5da1e-a678-492c-b0c6-7e5728c6c745)

Reply:
Smaia and Ssaia are clearly defined in the AIA spec. This does not include things like the APLIC. It should also be noted that many CSRs above are WARL, so zero is a valid implementation of some of them, making it very similar to the original behavior in machine mode.